while true do
print([[###....-++#++++--###
##+....-+++------###
##+....-+++-+--..###
##+.....-++---...###
##+....--+++--+++###
##+.-+++++++--######]])
wait(0.1)
print([[###......--+++---###
##+.....-++#++++-###
##+....-+++++++-.###
##+.....+++++--..###
##+.....-+++-+-..###
##+....--+++++-.-###]])
wait(0.1)
print([[###-...----------###
###....--++-----.###
###....--+------.###
###...-+++-++---.###
###..--##+++#----###
###----+##+##--..###]])
wait(0.1)
print([[###..-..------...###
###..-...-----...###
###..-..------...###
###..---+#++++...###
###..-++++----...###
###-----.........###]])
wait(0.1)
print([[###...------.....###
###...-------....###
###..--.----+-...###
###..---+#++--...###
###..--++----....###
###------........###]])
wait(0.1)
print([[###...-.+##+########
###...-.-#+++#######
##+...-.-+++--+#####
###...-.-####+######
###-.---+###########
###-----############]])
wait(0.1)
print([[###.........--..-###
###......-+---...###
###..-.----.-....###
###.--+++-++--...###
###-.-+##+++--...###
###---+##++++----###]])
wait(0.1)
print([[###.....---.-...-###
###.....---.--...###
###..--++----....###
###.--+##++---...###
###--++##+++--...###
###--++##++------###]])
wait(0.1)
end
